drogon_ctl/HandlerParser.cc

```diff
@@ -0,0 +1,168 @@
+#include "HandlerParser.h"
+#include <iostream>
+#include <regex>
+
+using namespace drogon::internal;
+
+std::pair<std::string, std::string> StructNode::findContentOfClassOrNameSpace(
```

Comment:
Would be good to have checks for comments here, it will break if the class or namespace includes commented uneven brackets

Reply:
Yes, you are right, thanks.

Reply:
If you want I can help you with that so you can focus on the main dev

Reply:
Great, thank you very much, I look forward to your PR :)